### PR TITLE
Damienbod/angular to angularjs

### PR DESCRIPTION
**Pull request by damienbod**
Updating to ASP.NET 1.1 Core msbuild
Renaming Angular ro  AngularJS
Angular 1 => AngularJS
Angualr 2 & 4  => Angular

### CODE REVIEW

**Comment by damienbod** [comment]
@Rick-Anderson

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,6 @@ _build/
 _site/
 Properties/
 node_modules/
-wwwroot/
```
Comment (Rick-Anderson) [inline .gitignore]: We'd like to keep the default of ignoring wwwroot then overriding in the few cases you need that directory

File: .gitignore [hunk 2]
```diff
@@ -213,4 +212,6 @@ project.lock.json
 __pycache__/
 
 #Mac OSX 
-.DS_Store
\ No newline at end of file
+.DS_Store
+aspnetcore/client-side/angular/sample/AngularJSSample/.vs/AngularJSSample/v15/.suo
```
Comment (Rick-Anderson) [inline .gitignore]: Just add .vs and .suo  in the Visual Studio section.

Comment (damienbod) [inline .gitignore]: removed them, thanks

File: .gitignore (hunk 1 above)
Comment (damienbod) [inline .gitignore]: added it back

**Pull request merged**